### ISSUE TITLE
ERROR: 2017 Image Fatal Error after Docker update to 2.3.0.2

### ISSUE THREAD

**Issue by weisso5** [microsoft/mssql-docker#603]
Upgraded to Docker Desktop 2.3.02 on Windows 10.

**Versions**

OS: Windows 10 Pro 64 Bit 1909 (18363.778)
Docker desktop version: 2.3.0.2 (45183)
Engine: 19.03.8
Compose: 1.25.5
Kubernetes: 1.16.5

**Run Command**

`docker run --name sql2017 -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=XXXXXXX' -v C:\Docker\Data\mssql:/var/opt/mssql -p 1433:1433 -d mcr.microsoft.com/mssql/server:2017-latest`

Result:

`This program has encountered a fatal error and cannot continue running at Thu May 14 02:33:51 2020 The following diagnostic information is available:           Reason: 0x00000007          Status: 0xc0000001         Message: Failed to load persistent registry. status code: c0000001      Stack Trace:                  file://package4/windows/system32/sqlpal.dll+0x000000000033D3A7                  file://package4/windows/system32/sqlpal.dll+0x000000000033C9F0                  file://package4/windows/system32/sqlpal.dll+0x000000000033A647                  file://package4/windows/system32/sqlpal.dll+0x000000000033A7C4                  file://package4/windows/system32/sqlpal.dll+0x0000000000339623                  file://package4/windows/system32/sqlpal.dll+0x0000000000338510                  file://package4/windows/system32/sqlpal.dll+0x000000000037E531         Process: 9 - sqlservr          Thread: 13 (application thread 0x4)     Instance Id: 640b19e0-fd48-411c-9e23-f8bfd1ed91ce        Crash Id: 9a73cf5d-67c5-46e1-8e34-c84564413b00     Build stamp: 5fe1db1037bde241cd5c20910433a2278daf1e28f58fcfd0640f3a3d2fd5e6cd    Distribution: Ubuntu 16.04.6 LTS      Processors: 2    Total Memory: 6239789056 bytes       Timestamp: Thu May 14 02:33:51 2020      Last errno: 38 Last errno text: Function not implemented  Ubuntu 16.04.6 LTS Capturing core dump and information to /var/opt/mssql/log... dmesg: read kernel buffer failed: Operation not permitted`

**Comment by sirajka** [comment]
I'm experiencing the same error in our AWS Kubernetes cluster.
I've tried several container images from docker hub (RHEL, Ubuntu, 2017, 2019) and all refuse to start.

Is there a fix or workaround? 

```
This program has encountered a fatal error and cannot continue running at Fri May 22 13:09:57 2020
The following diagnostic information is available:

         Reason: 0x00000002
     Parameters: 0x2002af9b0
                  [0] 0xffffffffc0000017
    Stack Trace:
                 file://package4/windows/system32/sqlpal.dll+0x000000000033D3A7
                 file://package4/windows/system32/sqlpal.dll+0x000000000033C8D0
                 file://package4/windows/system32/sqlpal.dll+0x000000000026694D
                 file://package4/windows/system32/sqlpal.dll+0x0000000000203243
                 file://package4/windows/system32/sqlpal.dll+0x000000000037CC08
                 file:///windows/system32/ntdll.dll+0x00000000000CBD48
                 file:///windows/system32/ntdll.dll+0x00000000000C36C5
                 file:///windows/system32/ntdll.dll+0x00000000000750DB
                 file:///windows/system32/ntdll.dll+0x000000000007508E
                 file://package4/windows/system32/sqlpal.dll+0x000000000024D837
                 file://package4/windows/system32/sqlpal.dll+0x000000000037E531
                 <unknown>+0x000000009C3FAA00
        Process: 14 - sqlservr
         Thread: 28 (application thread 0x2c)
    Instance Id: f9b095b7-9252-4ca8-95ad-aa49b445f951
       Crash Id: 790ba87a-c5bf-4b65-8127-bbfba2395d03
    Build stamp: 5fe1db1037bde241cd5c20910433a2278daf1e28f58fcfd0640f3a3d2fd5e6cd
   Distribution: Ubuntu 16.04.6 LTS
     Processors: 4
   Total Memory: 16423165952 bytes
      Timestamp: Fri May 22 13:09:57 2020

Ubuntu 16.04.6 LTS
Capturing core dump and information to /var/opt/mssql/log...
No journal files were found.
No journal files were found.
Fri May 22 13:09:58 UTC 2020 Capturing program information
Fri May 22 13:09:58 UTC 2020 Attempting to capture a dump with paldumper for pid 14
Captured a dump with paldumper
Fri May 22 13:09:59 UTC 2020 Capturing program binaries
Fri May 22 13:09:59 UTC 2020 Compressing the dump files
Core dump and information are being compressed in the background. When
complete, they can be found in the following location:
  /var/opt/mssql/log/core.sqlservr.05_22_2020_13_09_57.14.tbz2
```

**Comment by samwatts98** [comment]
Yeah I'm also repeatedly experiencing this issue. Following the instructions on [the Microsoft Docker SQL Server tutorial](https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-docker?view=sql-server-ver15&pivots=cs1-bash), I get the following stacktrace:

`SQL Server 2019 will run as non-root by default.

This container is running as user root.

To learn more visit https://go.microsoft.com/fwlink/?linkid=2099216.

This program has encountered a fatal error and cannot continue running at Sun May 24 16:58:08 2020

The following diagnostic information is available:


Reason: 0x00000007

Status: 0xc0000001

Message: Failed to load persistent registry. status code: c0000001

Stack Trace:

file://package4/windows/system32/sqlpal.dll+0x000000000033D3A7

file://package4/windows/system32/sqlpal.dll+0x000000000033C9F0

file://package4/windows/system32/sqlpal.dll+0x000000000033A647

file://package4/windows/system32/sqlpal.dll+0x000000000033A7C4

file://package4/windows/system32/sqlpal.dll+0x0000000000339623

file://package4/windows/system32/sqlpal.dll+0x0000000000338510

file://package4/windows/system32/sqlpal.dll+0x000000000037E531

Process: 8 - sqlservr

Thread: 12 (application thread 0x4)

Instance Id: 54bf49b0-f076-4d4b-8e8f-d90db22f58aa

Crash Id: 33fc7e33-19b5-4ef0-8256-5f53a8fba017

Build stamp: 5fe1db1037bde241cd5c20910433a2278daf1e28f58fcfd0640f3a3d2fd5e6cd

Distribution: Ubuntu 16.04.6 LTS

Processors: 2

Total Memory: 2088132608 bytes

Timestamp: Sun May 24 16:58:08 2020

Last errno: 38

Last errno text: Function not implemented

Ubuntu 16.04.6 LTS

Capturing core dump and information to /var/opt/mssql/log...

dmesg: read kernel buffer failed: Operation not permitted

No journal files were found.

Sun May 24 16:58:09 UTC 2020 Capturing program information

Sun May 24 16:58:09 UTC 2020 Attempting to capture a dump with paldumper for pid 8

Captured a dump with paldumper

Sun May 24 16:58:12 UTC 2020 Capturing program binaries

Sun May 24 16:58:13 UTC 2020 Compressing the dump files`

**Comment by madorin** [comment]
I have the same issue on some machines. On Win10 LTSC it works.

**Comment by Riposati** [comment]
Thanks for the solution.
